pom.xml

```diff
@@ -1674,7 +1674,7 @@
       <plugin>
         <groupId>org.codehaus.mojo</groupId>
         <artifactId>taglist-maven-plugin</artifactId>
-        <version>2.4</version>
+        <version>3.0.0</version>
         <configuration>
           <!-- skip till http://jira.codehaus.org/browse/MTAGLIST-71 -->
```

Comment:
@romani according to release notes, issue track seems to have moved. You want to check if this `till` tag needs to be updated or recreated.

Reply:
https://www.mojohaus.org/taglist-maven-plugin/issue-management.html

https://github.com/mojohaus/taglist-maven-plugin/issues/

Reply:
We can remove comment, they did not migrate issues, so we lost reason

Reply:
https://github.com/mojohaus/taglist-maven-plugin/issues/7 osed without fix